### PR TITLE
Dylan Zocchi

### PR DESCRIPTION
**Pull request by DylanZocchi**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] A CSS file is effectively linked to the HTML file in the Head section: 15 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 15 points 
- [ ] Appropriate use of Semantic Elements: 15 points
- [ ] Effective use of box model: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 25 points

**Checkpoint 2**
- [ ] Effective use of Flexbox layout: 15 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 15 points 
- [ ] Effective use of Bootstrap Grid: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Site is hosted on github: 15 points
- [ ] The student demonstrated mastery of HTML/CSS and Responsive Web Design by building a website of professional caliber: 25 points

**Checkpoint 3**
- [ ] Use of HTML: 15 points
- [ ] Use of CSS: 15 points
- [ ] Use of JQuery: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] The application has a professional look: 15 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

File: 03week/Checkpoint/index.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Grocery Delivery - Shipt</title>
+    <link rel="stylesheet" href="..reset.css">
```
Comment (kevincolten) [inline 03week/Checkpoint/index.html]: needs to be `../../reset.css`

**Comment by DylanZocchi** [comment]
I did realize this,  a bit too late though.    Was on the hustle to get the project done in time (took me a while) and assumed if I corrected it I would have a whole new issue to deal with.  I think I’ll just copy and paste it into my css folder for each project from now on.  The ../ thing kinda confuses me even though I think I get it.  I had all images in one folder and had to use ./ for some and ../ for others which perplexed me, since they were in the same spot.  

Was this only issue with the checkpoint?

Holla back.

**Comment by kevincolten** [comment]
the `./` and the `../` are how you can navigate from the file you are to another file. So the amount of `../` will change based on what file you are currently in.

yep

**Comment by DylanZocchi** [comment]
Hey Kevin.  Just noticed I got a 50 on the Tomorrows Tech assignment.  What was the issue with the assignment?  Am I not getting something wrong with the media queries?

**Comment by DylanZocchi** [comment]
I updated the l project.  I noticed I forgot to add links and the social media icons on the bottom.  I apologize for that.  Caught a nasty bug last week, was pretty much down for the count and dusted on robitussin  all weekend.  Other than that I thought I had everything working and the page looking good.  Let me know if theres something I’m missing or not getting.

**Pull request closed without merging**